### PR TITLE
Added tests for unordered list

### LINKED ISSUE #31: Backfill tests for Molecules/UnorderedList
Currently there is only a single default props "render-style" test

Using other component tests as a guide, write automated tests to cover important scenarios in component. Try to avoid granular assertions of the markup. Stay as high-level as possible to avoid test fragility.

### PR DESCRIPTION
**Pull request by SaidShah**
- [x] Related GitHub issue(s) linked in PR description
- [x] Destination branch merged, built and tested with your changes
- [x] Code formatted and follows best practices and patterns
- [x] Code builds cleanly (no _additional_ warnings or errors)
- [x] Manually tested
- [x] Automated tests are passing
- [x] No _decreases_ in automated test coverage
- [x] Documentation updated (readme, docs, comments, etc.)
- [x] Localization: No hard-coded error messages in code files (_minimally_ in string constants)
- [x] New component and/ or properties? Make sure to update storybook

Closes #31

### CODE REVIEW

**Review by jstevenson — APPROVED** [review]

**Comment by SaidShah** [comment]
@jstevenson I did change it, thank you for the input👍.

**Review by jstevenson — APPROVED** [review]

File: src/molecules/lists/unordered-list.tsx [hunk 1]
```diff
@@ -2,6 +2,12 @@ import React from "react";
 import { Icons } from "../../atoms/constants/icons";
 import { StringUtils } from "andculturecode-javascript-core";
 
+// -------------------------------------------------------------------------------------------------
+// #region Constants
+// -------------------------------------------------------------------------------------------------
+
+export const UnorderedListIconClassName = "-has-icon";
```
Comment (brandongregoryscott) [inline src/molecules/lists/unordered-list.tsx:9]: Looks like this is missing an endregion tag

**Review by brandongregoryscott — APPROVED** [review]
Approving this (and following PRs to backfill tests) for now, but I'll wait on merging all of them until we decide on whether to pull in `@testing-library/jest-dom` for the extended matchers.

**Comment by brandongregoryscott** [comment]
> Is there a reason for using faker rather than just using a static class name without spaces? This is less readable, is slower, uses more memory, and as your update shows, can be wrong.

While it's not something that _really_ needs to be randomized, as a general rule of thumb, if it doesn't _need_ to be a specific value, I prefer to keep the data randomized - it helps shed light on edge cases that might not be handled and to be honest, we use faker all over the place - not sure we're really concerned about memory usage for unit testing at this scale.

File: src/molecules/lists/unordered-list.tsx (hunk 1 above)
Comment (SaidShah) [reply src/molecules/lists/unordered-list.tsx:9]: @brandongregoryscott Thanks for catching that. I added the endregion tag 👍

**Review by wintondeshong — APPROVED** [review]

**Pull request merged**